MonMaraicher/Model/Farmer/Farmers.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Produceurs.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 10/04/2024.
+//
+
+import Foundation
+
+struct Farmers: Decodable, Hashable {
+    let nbTotal: String
+    let items: [Farmer]
+}
+
+struct Farmer: Decodable, Identifiable, Hashable {
+    let id: Int
+    let raisonSociale: String
+    let telephone: String?
+//    let email: String?
```

Comment:
Your PR should not contain dead code (code in comment)

Reply:
Yes, I know, I should have put a global comment to explain that it is being displayed and that I am gradually reducing the list of information